tools/oav-traffic-converter/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "oav-traffic-converter",
```

Comment:
If this is going to npm we should consider `@azure-tools` as the scope.

Reply:
100%. @praveenkuttappan called me out too and we resolved it 👍